### ISSUE TITLE
What about changing default preset directory?

### ISSUE THREAD

**Issue by kokoko3k** [Starman99x/koko-aio-presets-and-overlays#1]
Hi there,<br>
I noticed that you're using something like:
```#reference "../../../../shaders_slang/bezel/koko-aio/Presets-4.1/tv-aperturegrille-bloom-bezel.slangp"```

However, this requires your presets to be used exclusively with koko-aio shipped with Retroarch.

However, you could modify the line in:
```#reference "../../../../Presets-4.1/tv-aperturegrille-bloom-bezel.slangp"```

That way, users can put your koko-aio-packs directory under koko-aio-whatever and it would work for Retroarch shipped shaders as well as the development repo one:

![image](https://github.com/Starman99x/koko-aio-presets-and-overlays/assets/1230726/74bfd39c-b8be-463b-9d97-0567d9e44476)

What do you think?

**Comment by Starman99x** [comment]
You're right. I kept them that way because I only usually test the koko-aio shader that comes shipped with RetroArch. Since it's more stable.

**Comment by kokoko3k** [comment]
Yeah, i noticed while trying to debug the flickering issue :)
